packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
-  - package: snowplow/snowplow_utils
-    version: [">=0.13.0", "<0.14.0"]
+ - git: "https://github.com/snowplow/dbt-snowplow-utils.git"
```

Comment:
This should be a separate commit imo

Reply:
It was, until rebase hell, if we release utils before we merge this it'll get removed and squashed.

Reply:
temporary separate commit added